examples/controls_example/public/react_controls/timeslider_control/get_timeslider_control_factory.tsx

```diff
@@ -185,8 +185,7 @@ export const getTimesliderControlFactory = (
       const viewModeSubject =
         getViewModeSubject(controlGroupApi) ?? new BehaviorSubject('view' as ViewMode);
 
-      // overwrite the `width` attribute because time slider should always have a width of large
-      const defaultControl = initializeDefaultControlApi({ ...initialState, width: 'large' });
+      const defaultControl = initializeDefaultControlApi(initialState);
```

Comment:
We need this to be overwritten with `large` still, no? This is the way it works on `main` - the time slider always has a width of `large`

Reply:
resolved with da087d86e8a6fdb1ab6411f18fe39f19f994c19f